oxenc/bt_serialize.h

```diff
@@ -912,6 +946,8 @@ class bt_list_consumer {
         if (data.size() != 1)
             throw bt_deserialize_invalid{"Dict finished without consuming the entire buffer"};
     }
+
+    using value_type = bt_value;
```

Comment:
I added this so it would be workable with concepts that use `::value_type`. Also maybe I'm misunderstanding, but it still accesses that type in release builds?